index.html

```diff
@@ -5007,13 +4980,254 @@ <h3 id="looping-AudioBufferSourceNode">
+            <li>The playback of a looped buffer should behave identically to an
+            unlooped buffer containing consecutive occurrences of the looped
+            audio content, excluding any effects from interpolation.
+            </li>
+          </ul>
+          <p>
+            The description of the algorithm is as follows:
+          </p>
+          <ol>
+            <li>Let <code>start</code> correspond to the <code>when</code>
+            parameter to <code>start()</code> if supplied, else 0.
+            </li>
+            <li>Let <code>stop</code> correspond to the sum of the
+            <code>when</code> and <code>duration</code> parameters to
+            <code>start()</code> if supplied, or the <code>when</code>
+            parameter to <code>stop()</code>, otherwise <em>Infinity</em>.
+            </li>
+            <li>Let <code>buffer</code> be the <a>AudioBuffer</a> employed by
+            the node.
+            </li>
+            <li>Let <code>offset</code> correspond to the <code>offset</code>
+            parameter to <code>start()</code> if supplied, else 0. This is the
+            <a>playhead position</a> in the buffer at which playback will
+            begin.
+            </li>
+            <li>Let <code>loop</code>, <code>actualLoopStart</code> and <code>
+              actualLoopEnd</code> be the loop-related attributes of the node,
+              as discussed in the preceding section on looping.
+            </li>
+            <li>Let the function <code>playbackSignal(position)</code> be the
+            <em>playback signal function</em> for <code>buffer</code>, which is
+            a function that maps from a <a>playhead position</a> to a set of
+            output signal values, one for each output channel. If
+            <code>position</code> corresponds to the location of an exact
+            sample frame in the buffer, this function returns that frame.
+            Otherwise, its return value is determined by a UA-supplied
+            algorithm that interpolates between sample frames in the
+            neighborhood of <code>position</code>. The UA may also use
+            resampled copies of the original buffer as the basis for
+            interpolation, as per UA-specific optimization strategies. If
+            <code>position</code> is greater than or equal to
+            <code>actualLoopEnd</code> and there is no subsequent sample frame
+            in <code>buffer</code>, then the next sample frame should be taken
+            as the playback signal at <code>actualLoopStart</code> for
+            interpolation purposes.
+            </li>
+            <li>Let <code>bufferTime</code> equal -1. This is the <em>playhead
+            position</em> within <code>buffer</code> of the next output sample
+            frame. The initial value -1 indicates that this position has not
+            yet been determined.
+            </li>
+            <li>For each block of audio to be rendered:
```

Comment:
I'll leave the decision to you. I just thought code would be easier to read and understand at this point because there's nothing that can't be easily expressed in JS.

I'm ok with `playbackSignal()` not being in JS.  I'm ok if you added, say, `output(sampleFrame,index)` to output magically the output of `playbackSignal()` at the given `index`.  Perhaps that's not appropriate for a spec; I don't know.